rms/search_engine/clients.py

```diff
@@ -71,3 +74,59 @@ def _build_abstract_keywords_criteria(abstract_keywords: list[str] | None) -> st
             return ""
         abstract_keywords_criteria = " OR ".join([f'ABS("{keyword}")' for keyword in abstract_keywords])
         return f"({abstract_keywords_criteria})"
+
+
+class DblpClient(ABC):
+
+    def __init__(self):
+        self.settings = Settings()
+        self.headers = {
+            "httpAccept": "application/json",
+        }
+        self.dblp_search_api_endpoint = self.settings.dblp_search_endpoint
+
+
+class DblpAuthorSearchApi(DblpClient):
+
+    def __init__(self):
+        super().__init__()
```

Comment:
Not needed when initialization behavior is not changed.

Reply:
fixed